dubbo-remoting/dubbo-remoting-netty4/src/test/java/org/apache/dubbo/remoting/transport/netty4/ReplierDispatcherTest.java

```diff
@@ -14,93 +14,120 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.dubbo.remoting;
+package org.apache.dubbo.remoting.transport.netty4;
 
 import org.apache.dubbo.common.URL;
+import org.apache.dubbo.common.constants.CommonConstants;
+import org.apache.dubbo.common.utils.NetUtils;
+import org.apache.dubbo.remoting.RemotingException;
 import org.apache.dubbo.remoting.exchange.ExchangeChannel;
+import org.apache.dubbo.remoting.exchange.ExchangeServer;
 import org.apache.dubbo.remoting.exchange.Exchangers;
-import org.apache.dubbo.remoting.exchange.support.Replier;
 import org.apache.dubbo.remoting.exchange.support.ReplierDispatcher;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import java.io.Serializable;
 import java.util.Random;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.Executor;
-import java.util.concurrent.Executors;
+import java.util.concurrent.*;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
 
 /**
- * Main
+ * ReplierDispatcherTest
  */
 
-public class Main {
-    public static void main(String[] args) throws Exception {
-        startServer(9010);
-        mutliThreadTest(10, 9010);
-        dataPackageTest(9010);
-    }
+public class ReplierDispatcherTest {
+
+    private ExchangeServer exchangeServer;
+
+    private ConcurrentHashMap<String, ExchangeChannel> clients = new ConcurrentHashMap<>();
 
-    private static void startServer(int port) throws Exception {
+    private int port;
+
+    @BeforeEach
+    public void startServer() throws RemotingException {
+        port = NetUtils.getAvailablePort();
         ReplierDispatcher dispatcher = new ReplierDispatcher();
         dispatcher.addReplier(RpcMessage.class, new RpcMessageHandler());
-        dispatcher.addReplier(Object.class, new Replier<Object>() {
-            public Object reply(ExchangeChannel channel, Object msg) {
-                for (int i = 0; i < 10000; i++)
-                    System.currentTimeMillis();
-                System.out.println("handle:" + msg + ";thread:" + Thread.currentThread().getName());
-                return new StringMessage("hello world");
+        dispatcher.addReplier(Data.class, (channel, msg) -> {
+            for (int i = 0; i < 10000; i++) {
```

Comment:
I know this is existing code. but I'm thinking what's purpose for this code? looks like this is not necessary for this unit test

Reply:
Yes, this loop seems to be unnecessary, just to keep the existing code. I have changed it.